pipeline/alert.go

```diff
@@ -733,6 +737,65 @@ type HipChatHandler struct {
 	Token string
 }
 
+// Send the alert to JIRA.
```

Comment:
Fixed with https://github.com/influxdata/kapacitor/pull/644/commits/22854adb636f6bc51a7a22b760ce06d517973376. I think that should be clear enough.